examples/flask_app/app.py

```diff
@@ -0,0 +1,54 @@
+import json
+import pcf
+
+from flask import Flask, request
+from pcf.core import State
+from pcf.quasiparticle.aws.ec2_route53.ec2_route53 import EC2Route53
+
+app = Flask(__name__)
+
+#We read the json configuration of our desired quasiparticle
+with open('pcf_example_config.json', 'r') as f:
+    pcf_config = json.load(f)
+
+#Here we initialize the desired state definitions
+ec2_route53_quasiparticle.set_desired_state(State.running)
+
+#GET /pcf endpoint returns the current state of our EC2 Route53 Quasiparticle
+@app.route('/pcf', methods=['GET'])
+def get_pcf_status():
+    return str(ec2_route53_quasiparticle.get_state())
```

Comment:
ec2_route53_quasiparticle not defined here...should create definition with only unique identifier to get state.